tidb-cloud/tidb-cloud-billing-dm.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Data Migration Billing
+summary: Learn about billing for Data Migration in TiDB Cloud.
+---
+
+# Data Migration Billing
+
+This document describes the billing for Data Migration in TiDB Cloud.
+
+## Specifications for Data Migration
+
+| Specification | Full data migration | Incremental data migration |
+|---------------|---------------------|----------------------------|
+| 2  | 25 MiB/s | 10 k [^*] |
+| 4  | 35 MiB/s | 20 k |
+| 8  | 40 MiB/s | 40 k |
+| 16 | 45 MiB/s | 80 k |
+
+[^*]: this unit indicates the number of rows per second generated by incremental replication Data Manipulation Language (DML) statements. For example, 10 k means 10000 rows per second.
```

Comment:
10 k means 10000 rows per second and also means if the upstream insert \update\delete within 10000 rows,the dm job can sync the 10000 rows to the downstream within 1 second.